packages/clerk-js/src/ui/components/OrganizationProfile/InviteMembersForm.tsx

```diff
@@ -75,8 +72,7 @@ export const InviteMembersForm = (props: InviteMembersFormProps) => {
 
   const roleField = useFormControl('role', 'basic_member', {
     options: roles,
-    // label: localizationKeys('formFieldLabel__firstName'),
-    // placeholder: localizationKeys('formFieldInputPlaceholder__firstName'),
+    // TODO: localize this
```

Comment:
What's the level of effort to localize these values? Do we have a task for it?

Reply:
The value is already localized, i updated the code to reflect the correct state.